File: Python Code/Daily_Flash_Code_0/Week11/4Apr/Python/AddSer/AddSer.py
import math as m
le=int(input("Enter length of series:"))
add=0
x,y=input("Enter Value of X and Y:").split()
x=int(x)
y=int(y)
for i in range(1,le+1):
    add+=i/(x+y)**i *m.factorial(i)
print("The Addition of entered series:",round(add,2))

File: Python Code/Daily_Flash_Code_0/Week12/6Apr/C Code/AddSer.c
#include<stdio.h>
#include<math.h>
//double pow(double,double);
int fact(int x){
int fa=1;
for(int i=1;i<=x;i++)
        fa*=i;
return fa;
}

void main(){
int x,y;
printf("Enter X and Y:");
scanf("%d%d",&x,&y);

int le;
printf("Enter Length of series:");
scanf("%d",&le);

int add=0;
for(int i=1;i<=le;i++){
	x=fact(i);
	printf("X=%d",x);
	add+=(pow((x+y),i)+x);
	printf("Add=%d\n",add);
}
printf("%d\n",add);
}

